src/react-components/layout/Toolbar.scss

```diff
@@ -17,16 +18,25 @@
   display: flex;
   flex: 1;
   justify-content: space-around;
+  align-items: center;
+  gap: 0 24px;
 
   @media (min-width: theme.$breakpoint-lg) {
     flex: 0;
+  }
+}
 
-    & > * {
-      margin-right: 24px;
-    }
+:local(.centerContent) {
```

Comment:
if you can somehow get rid of the camelCase here that would be awesome. If it's baked in, no big.